src/pages/workspace/expensifyCard/WorkspaceCardPageFeed.tsx

```diff
@@ -0,0 +1,37 @@
+import type {StackScreenProps} from '@react-navigation/stack';
+import React from 'react';
+import type {OnyxEntry} from 'react-native-onyx';
+import type {FullScreenNavigatorParamList} from '@libs/Navigation/types';
+import AccessOrNotFoundWrapper from '@pages/workspace/AccessOrNotFoundWrapper';
+import CONST from '@src/CONST';
+import type SCREENS from '@src/SCREENS';
+import type {WorkspaceCardsList} from '@src/types/onyx';
+import {isEmptyObject} from '@src/types/utils/EmptyObject';
+import WorkspaceCardPageEmptyState from './WorkspaceCardPageEmptyState';
+import WorkspaceExpensifyCardPage from './WorkspaceExpensifyCardPage';
+
+type WorkspaceCardPageFeedProps = StackScreenProps<FullScreenNavigatorParamList, typeof SCREENS.WORKSPACE.EXPENSIFY_CARD>;
+
+// TODO: remove when Onyx data is available, and pass the data to 'WorkspaceExpensifyCardPage' so that we will not make the same 'Onyx' call twice
+const cardsList: OnyxEntry<WorkspaceCardsList> = {};
+
+function WorkspaceCardPageFeed({route}: WorkspaceCardPageFeedProps) {
```

Comment:
I wonder if this should be called `WorkspaceExpensifyCardPage`. Then WorkspaceExpensifyCardPage below could be called WorkspaceExpensifyCardListPage to make it more clear what each is doing

Reply:
+1 to the suggestion from @MariaHCD

Reply:
ok